### ISSUE TITLE
Icons are stretched in servo

### ISSUE THREAD

**Issue by mdinger** [servo/servo#7753]
The people icons at the top on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/padding) are stretched in servo.

[Testcase1](https://mdinger.github.io/servo_7753/testcase1.html):

``` html
<html>
<head>
<style>
    img {
        height: auto!important;
        max-width: 20px;
    }
</style>
</head>
<body>
    <img src="testcase1_files/head.jpeg" height="34" width="34">
</body></html>
```

---

![stretched_icon](https://cloud.githubusercontent.com/assets/4156987/10121171/1dedc720-64ad-11e5-9aa6-63eb332c0a40.png)

**Comment by mdinger** [comment]
Updated with linked testcase

**Comment by jdm** [comment]
Looks like our prioritisation of image content attributes vs. CSS properties is not working correctly.

**Comment by mdinger** [comment]
Oh, the servo build I used was version 4dc986bca3 so it's a few days old. The MDN docs also have quite a few things wrong with them.

**Comment by mdinger** [comment]
With 31657811d33f23c79eae49dea171718562b031c5, this is still a problem.

**Comment by stshine** [comment]
This is fixed by #14490.

**Comment by jdm** [comment]
Thanks!
